vspace/src/main/java/edu/asu/diging/vspace/web/staff/ExhibitionConfigurationController.java

```diff
@@ -38,17 +38,24 @@ public class ExhibitionConfigurationController {
     @Autowired
     private ExhibitionFactory exhibitFactory;
 
+    public static final String EXH_PREVIEW = "EXH_PREVIEW_";
+    
     @RequestMapping("/staff/exhibit/config")
     public String showExhibitions(Model model) {
         // for now we assume there is just one exhibition
-        IExhibition exhibition = exhibitManager.getStartExhibition();
-        if(exhibition!=null) {
-            model.addAttribute("exhibition", exhibition);
+        Exhibition exhibition = (Exhibition) exhibitManager.getStartExhibition();
+        String previewId = null;
+        if (exhibition == null) {
+            exhibition = (Exhibition) exhibitFactory.createExhibition();
         } else {
-            model.addAttribute("exhibition", new Exhibition());
+            previewId = exhibition.getPreviewId();
+            if(previewId==null || previewId.isEmpty()) {
+                exhibitFactory.updatePreviewId(exhibition);
```

Comment:
I would do this in the `getStartExhibition` method so not every place where we might need the preview id this needs to be done.